tileview/src/main/java/com/qozix/tileview/tiles/Tile.java

```diff
@@ -257,7 +262,7 @@ void reset() {
     }
     mState = State.UNASSIGNED;
     mRenderTimeStamp = null;
-    if( mBitmap != null && !mBitmap.isRecycled() ) {
+    if( mBitmap != null ) {
       BitmapRecycler recycler = mBitmapRecyclerReference.get();
```

Comment:
In the case of the `execute( TileRenderPoolExecutor )` call, `mBitmapRecyclerReference` is null. So we get a `NullPointerException` here.